src/main/kotlin/no/nav/syfo/kartleggingssporsmal/application/KartleggingssporsmalService.kt

```diff
@@ -56,7 +52,8 @@ class KartleggingssporsmalService(
 
     companion object {
         private val log = LoggerFactory.getLogger(KartleggingssporsmalService::class.java)
-        private const val PILOTKONTOR_TEST = "0314"
-        private val pilotkontorer = listOf(PILOTKONTOR_TEST)
+        private const val KONTOR_NAV_LIER = "0626"
+        private const val KONTOR_NAV_ASKER = "0220"
```

Comment:
Betyr også at testing i dev fremover må være på sykmeldte som bor her. Jeg har lagt til tilganger for disse kontorene i Ida slik at jeg får opp oversikten for kontorene, og så får vi lage noen dolly-personer (evt rute noen dit 😎) som bor der.